src/types.ts

```diff
@@ -18,6 +18,7 @@ export interface Package {
   peerDependencies?: {
     [name: string]: string;
   };
+  license?: string;
```

Comment:
I don't think this is optional.

Reply:
changed it to be compulsory and the code to match.